MPC-PDA/text.tex

```diff
@@ -330,128 +330,149 @@ \section{Rekurentní neuronové sítě. LSTM. UNet sítě. Struktury neuronovýc
+Dále pro každý vrchol $u$ se sesbírá množina navštívených vrcholů pomocí na~cestě (pomocí random-walk strategie) začínajících z~$u$.
+Nakonec se optimalizuje embeddings (pro vrchol $u$ dokážeme předpovědět sousedy v~množině).
 
-\[\mathcal{L} = \sum_{u\in V} \sum_{v\in N_R (u)} - \log(\frac{\exp(\mathbf{z}_{u}\,\mathbf{z}_{v})}{\sum_{n\in V} \exp(\mathbf{z}_{u}\,\mathbf{z}_{n})})\]
+$$\mathcal{L} = \sum_{u\in V} \sum_{v\in N_R (u)} - \log(\frac{\exp(\mathbf{z}_{u}\,\mathbf{z}_{v})}{\sum_{n\in V} \exp(\mathbf{z}_{u}\,\mathbf{z}_{n})})$$
 
-První suma značí součet přes všechny vrcholy \textit{u}.
-Druhá suma značí součt přes vrcholy \textit{v} vyskytujících se na cestě z vrcholu \textit{u}.
-Obsah logaritmu značí předpovídanou pravděpodobnost společného výskytu \textit{u} a \textit{v} na cestě.
+První suma značí součet přes všechny vrcholy $u$.
+Druhá suma značí součt přes vrcholy $v$ vyskytujících se na~cestě z~vrcholu $u$.
+Obsah logaritmu značí předpovídanou pravděpodobnost společného výskytu $u$ a $v$ na~cestě.
 
 Cílem je najít $z_u$ které minimalizuje $\mathcal{L}$.
 
 
 \subsubsection{node2vec}
 
-Node2vec je velice podobný Deepwalk s tím rozdílem, že jsou jinak vybírany sousední vrcholy.
-Hlavní myšlenkou je kompromis mezi lokálním (BFS) a globálním (DFS) pohledem na graf (síť).
+Node2vec je velice podobný Deepwalk s~tím rozdílem, že jsou jinak vybírany sousední vrcholy.
+Hlavní myšlenkou je kompromis mezi lokálním (BFS) a globálním (DFS) pohledem na~graf (síť).
 Node2vec má dva parametry:
 \begin{itemize}
-	\item \textbf{p} udávajací hodnotu návratu k předchozímu vrcholu (return parameter)
-	\item \textbf{q} udavající \uv{poměr} mězi BFS a DFS (\uv{walk away} parameter)
+	\item $p$ udávajací hodnotu návratu k~předchozímu vrcholu (return parameter)
+	\item $q$ udavající \enquote{poměr} mězi BFS a DFS (\enquote{walk away} parameter)
 \end{itemize}
 
-Při zvolení $q>p$ se algoritmus zaměřuje spíše na globální pohled (bude směřovat k DFS průchodu). Při zvolení $q<p$ se algoritmus zaměří spíše na lokalní pohled (bude směřovat k BFS).
+Při zvolení $q>p$ se algoritmus zaměřuje spíše na~globální pohled (bude směřovat k~DFS průchodu). Při zvolení $q<p$ se algoritmus zaměří spíše na~lokalní pohled (bude směřovat k~BFS).
 
-\begin{center}
-	\includegraphics[width=\textwidth]{images/node2vecQandP.png}
-\end{center}
+\begin{figure}[ht]
+    \centering
+	\includegraphics[width=\textwidth]{images/11_node2vec-pq}
+    \caption{Prioritizace $p$ nebo $q$ v~node2vec}
+\end{figure}
 
-Při prochocházení grafem může dojít ke třem stavům (jsme v bodu w):
+Při prochocházení grafem (viz obr.~\ref{pruchod-node2vec}) může dojít ke třem stavům (z~hlediska bodu $w$):
 \begin{itemize}
-	\item vrátit se k předchozímu vrcholu (s1)
-	\item navštívit vedlejší vrchol, který má stejnou vzdálenost od počátečního vrcholu (s2)
-	\item navštivit vzdálenější vrchol (s3)
+	\item návrat se k~předchozímu vrcholu ($s1$),
```

Comment:
Zde asi odmazat `se` aby to dávalo smysl

Reply:
Fixed & force-pushed.